src/main/java/com/feedzai/commons/sql/abstraction/engine/impl/MySqlTranslator.java

```diff
@@ -229,6 +229,42 @@ public String translate(Query q) {
         return q.isEnclosed() ? ("(" + finalQuery + ")") : finalQuery;
     }
 
+    @Override
+    protected String translateUpdateFrom(final Update update) {
```

Comment:
We seem to have a couple of engines with proper UPDATE FROM support,
while others simply don't have it and need to use the MERGE statement.
This translator is kind of an outlier, I'd probably leave it using the default UpdateFrom with the MERGE statement,
but I guess this is also fine...